docs/source/app_developers_guide/ubuntu.rst

```diff
@@ -505,12 +505,5 @@ The output of the command will be similar to this:
   DATA: "b'\xa1fcCTdcH\x192B'"
   HEAD: "0c4364c6d5181282a1c7653038ec9515cb0530c6bfcb46f16e79b77cb524491676638339e8ff8e3cc57155c6d920e6a4d1f53947a31dc02908bcf68a91315ad5"
 
-
-Stopping Sawtooth Components
-============================
-
-To stop the validator, enter CTRL-c several times in the validator's terminal
-window.
-
-Stop the REST API and transaction processors by entering a single CTRL-c in
-the appropriate windows.
```

Comment:
Did you intend to delete this block?

Reply:
Looks like a mistake on my part after a rebase - will fix